src/petals/server/throughput.py

```diff
@@ -129,14 +135,17 @@ def measure_compute_rps(
     dtype: torch.dtype,
     *,
     load_in_8bit: bool,
+    tensor_parallel_devices: Sequence[torch.device],
     n_tokens: int = 16,
     n_steps: int = 500,
 ) -> float:
+    if not tensor_parallel_devices:
+        tensor_parallel_devices = (device,)
```

Comment:
The server does not enable tensor-parallelism if we have only one device. Please behave here similarly.

E.g., if tensor-parallelism breaks but the server works otherwise, this code will fail (even though the server would have worked).

Reply:
Discussion log:
- i complained that not wrapping with TensorParallel will complicate the code in backend.py, handler.py as follows:
```
If isinstance(backend.module, WrappedBloomBlock):
   do_thing()
elif isinstance(backend.module, TensorParallel):
   do_another_thing()
else:
   raise RuntimeError(...)
```

- i checked that TensorParallel with len(devices) <= 1 calls the model normally, without cross-device stuff
- @borzunov suggested that if we do wrap with TensorParallel, we should wrap the same way everywhere (e.g. server.py, throughput.py)